.github/workflows/minimum_python_versions.yaml

```diff
@@ -15,7 +15,7 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v1
+    - uses: actions/checkout@v2
```

Comment:
Is this really related to this PR?

Reply:
That was needed to make tests pass after a Github action failure, due to a GA bug. Not related to this PR.

Reply:
Github Action